cvprac/cvp_client.py

```diff
@@ -170,7 +171,7 @@ def set_log_level(self, log_level='INFO'):
         self.log.setLevel(getattr(logging, log_level))
 
     def connect(self, nodes, username, password, connect_timeout=10,
-                protocol='http', port=None):
+                port=None, cert=False):
```

Comment:
Mmm..not sure. I see why you did it (at least I think I do) based on the requirements for requests connect.
```
        :param verify: (optional) Either a boolean, in which case it controls whether we verify
            the server's TLS certificate, or a string, in which case it must be a path
            to a CA bundle to use. Defaults to ``True``.
```